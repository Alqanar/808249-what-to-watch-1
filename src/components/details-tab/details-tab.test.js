import React from "react";
import renderer from "react-test-renderer";

import DetailsTab from "./details-tab.tsx";


const filmMockData = {
  director: `Russo brothers`,
  famousActors: [`Robert Downey Jr.`, `Chris Hemsworth`, `Mark Ruffalo`],
  duration: 181,
  genre: [`Action`, `Adventure`, `Sci-Fi`],
  released: 2019
};

it(`Details tab is correctly renders`, () => {
  const tree = renderer
    .create(
      <DetailsTab
        film={filmMockData}
      />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
