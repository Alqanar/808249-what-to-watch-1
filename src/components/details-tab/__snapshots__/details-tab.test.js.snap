// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details tab is correctly renders 1`] = `
<div
  className="movie-card__text movie-card__row"
>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Director
      </strong>
      <span
        className="movie-card__details-value"
      >
        Russo brothers
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Starring
      </strong>
      <span
        className="movie-card__details-value"
      >
        Robert Downey Jr.,
        <br />
        Chris Hemsworth,
        <br />
        Mark Ruffalo,
        <br />
      </span>
    </p>
  </div>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Run Time
      </strong>
      <span
        className="movie-card__details-value"
      >
        3h 1m
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Genre
      </strong>
      <span
        className="movie-card__details-value"
      >
        Action, Adventure, Sci-Fi
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Released
      </strong>
      <span
        className="movie-card__details-value"
      >
        2019
      </span>
    </p>
  </div>
</div>
`;
